### ISSUE TITLE
Woodcutting LeafBlower help documentation is incorrect.

### ISSUE THREAD

**Issue by mibby** [mcMMO-Dev/mcMMO#4847]
https://github.com/mcMMO-Dev/mcMMO/blob/ddc9a69f4b19081dbd5fba84f96803341cb16f73/src/main/resources/locale/locale_en_US.properties#L971

Had someone notice this being incorrect. Default should be unlocked at level 150 in Retro mode, not 100. Not sure if the locale has any other skills unlock numbers incorrect.

Also noticed LeafBlower has Rank_2 and Rank_3 in skillranks but doesn't seem to be used for anything.

**Comment by Chew** [comment]
Good catch! Should be an easy PR for anyone who wants to fix it real quick.
